editor/project_manager/project_dialog.cpp

```diff
@@ -370,6 +370,8 @@ void ProjectDialog::_browse_project_path() {
 	} else {
 		fdialog_project->set_file_mode(EditorFileDialog::FILE_MODE_OPEN_DIR);
 	}
+
+	hide();
```

Comment:
This fixes the exclusive window error.

Reply:
IMHO you should also show the dialog when quit the broswer window?

Reply:
Done.